src/AfterDraf/index.ts

```diff
@@ -1,31 +1,56 @@
 import {Component} from 'react';
-
-const RAF = requestAnimationFrame;
+import {isClient} from '../util';
 
 export interface IAfterDrafState {
   ready: boolean;
 }
 
-export class AfterDraf extends Component<{}, IAfterDrafState> {
-  frame;
+const Passthrough = (props) => props.children;
 
-  state: IAfterDrafState = {
-    ready: false
-  };
+export const createAfterDraf = (times = 1) => {
```

Comment:
Can't think of a case were this factory will be necessary.

Reply:
Well, in your case for dealing with FOUC it is useful. See PR comment.

Reply:
the default component is enough to fix the FOUC.

Reply:
But not perfect. I will delay by DRAF on every mount. But you need to delay render only the very first mount.